ambry-api/src/main/java/com.github.ambry/config/CloudConfig.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2019 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.config;
+
+/**
+ * The configs for Cloud backup and recovery.
+ */
+public class CloudConfig {
+  /**
+   * Azure connection string.
+   */
+  @Config("azure.connection.string")
+  public final String azureConnectionString;
```

Comment:
We shouldn't have any azure specific things in a generic API class.
Also, the Azure destination won't be able to use it without the cosmosdb properties.
It might be hard to create a generic config class with named properties that will work for all providers.

Reply:
Azure/Aws and all other cloud based properties can be in this config class. People use the one they need. 
Another reason is AmbryLi depends on Config Classes to populate cfg2 configs. We need a config class.